lib/Core/SQLiteBuildDB.cpp

```diff
@@ -495,24 +526,44 @@ class SQLiteBuildDB : public BuildDB {
     assert(result == SQLITE_OK);
     result = sqlite3_step(insertIntoKeysStmt);
     if (result != SQLITE_DONE) {
-      *error_out = getCurrentErrorMessage();
-      return UINT64_MAX;
+      // FIXME: We need to support error reporting here, but the engine cannot
```

Comment:
We should probably have a radar for this considering we recently removed `abort` calls from the SQLite code.